doc/reporters/github-reporter.md

```diff
@@ -2,6 +2,20 @@
 
 Report about design and static code diagnostics issues in pull requests based on GitHub Actions Workflow commands.
 
+1. Your git workflow must include/support flutter/dart
+2. Add to your git workflow:
+```
```

Comment:
@furaiev please add empty line before backticks, and add language specification for quoted block